### PR TITLE
Update config parameters lr and grad_clip

### LINKED ISSUE #544: Ablation on lr vs. gradient norm clipping
### Is your feature request related to a problem? Please describe.

Question was whether we can increase the learning rate with a more conservative gradient norm clipping value. 

### Describe the solution you'd like

Run ablation experiments with following parameters:
- `lr_max: 5e-5, grad_clip: 5.0`
- `lr_max: 5e-5, grad_clip: 1.0`
- `lr_max: 2e-4, grad_clip: 5.0`
- `lr_max: 2e-4, grad_clip: 1.0`

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Organisation

ECMWF

### PR DESCRIPTION
**Pull request by Jubeku**
## Description

Ablations were run to test whether we can increase the learning rate with a more conservative gradient norm clipping value.
Results are posted in issue #544. 
After discussing within the ERA5 performance meeting, we concluded that a learning rate of 2e-4 might be slightly too big.
Should we at least double the learning rate and agree on `lr_max: 1e-4, grad_clip: 1.0`? 
cc @clessig @sophie-xhonneux 

## Type of Change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation update

## Issue Number

Closes #544 

## Code Compatibility

-   [ ] I have performed a self-review of my code

### Code Performance and Testing

-   [ ] I ran the `uv run train` and (if necessary) `uv run evaluate` on a least one GPU node and it works 
-   [ ] If the new feature introduces modifications at the config level, I have made sure to have notified the other software developers through Mattermost and updated the paths in the `$WEATHER_GENERATOR_PRIVATE` directory

### Dependencies

-   [ ] I have ensured that the code is still pip-installable after the changes and runs
-   [ ] I have tested that new dependencies themselves are pip-installable.
-   [ ] I have not introduced new dependencies in the inference portion of the pipeline

### Documentation

-   [ ] My code follows the style guidelines of this project
-   [ ] I have updated the documentation and docstrings to reflect the changes
-   [ ] I have added comments to my code, particularly in hard-to-understand areas

### CODE REVIEW

File: config/default_config.yml [hunk 1]
```diff
@@ -92,7 +92,7 @@ shuffle: True
 
 lr_scaling_policy: "sqrt"
 lr_start: 0.000001
-lr_max: 0.00005
+lr_max: 0.0002
```
Comment (sophie-xhonneux) [inline config/default_config.yml]: can you change the value as agree to 1e-4 and then we can merge
